### PR TITLE
contact form

### PR DESCRIPTION
**Pull request by dwolverton**

### CODE REVIEW

**Comment by dwolverton** [comment]
Nice form, bro!

File: style.css [hunk 1]
```diff
@@ -15,6 +16,8 @@ button, .button {
 .primary-button {
   background-color: dodgerblue;
   color: white;
+  border-style: outset;
+  border-color: dodgerblue;
```
Comment (dwolverton) [inline style.css:20]: Wait. Why did you include this change?

Comment (dwolverton) [reply style.css:20]: Because I was too lazy to create another branch. 🤷

File: index.html [hunk 2]
```diff
@@ -16,6 +16,22 @@ <h1>The Best Branch Demo!</h1>
...
+    <p>
+      <label for="name">Your name:</label>
+      <input type="text" id="name" name="name" />
+    </p>
+    <p>
+      <label for="message">Message:</label>
+      <textarea id="message" name="message"></textarea>
+    </p>
+    <p>
+      <button type="submit" class="primary-button">Send</button>
```
Comment (PunchesC) [inline index.html:31]: Great button!!

File: style.css [hunk 3]
```diff
@@ -2,8 +2,9 @@ body {
   margin-bottom: 60px;
 }
 
-h1 {
+h1, h2 {
   color: purple;
+  font-family: sans-serif;
```
Comment (dwolverton) [inline style.css:7]:
```suggestion
  font-family: Arial, sans-serif;
```

**Pull request merged**